_episodes/05-loops-and-functions.md

```diff
@@ -257,11 +257,11 @@ A more elegant solution would be to create a reusable tool that performs this
 task with minimum input from the user. To do this, we are going to turn the code
 we've already written into a function.
 
-Functions are reusable, self-contained pieces of code that are called with a
+Functions are reusable, self-contained pieces of code that can be called with a
 single command. They can be designed to accept arguments as input and return
 values, but they don't need to do either. Variables declared inside functions
 only exist while the function is running and if a variable within the function
-(a local variable) has the same name as a variable somewhere else in the code,
+(a local variable) has the same name as a variable somewhere else in the code (a global variable),
```

Comment:
Introducing the term "global variable" here without explaining it in more detail is confusing. If you'd really like this change accepted, please revise to clarify the meaning of the term "global variable".